web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1713,6 +1713,8 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
 
   @volatile var cometGetTimeout = 140000
 
+  @volatile var putAjaxFnsInContainerSession = false
```

Comment:
@farmdawgnation I'm not sure what happened in my head, but when I first saw your snippet I misunderstood how it would work for this setting. I get it now, lol. I opted for the approach @Shadowfiend used tho.  Which do y'all think is better?